apps/dashboard/src/components/layout/app-shell/index.tsx

```diff
@@ -19,6 +18,7 @@ import {
   TrackedLink,
 } from "tw-components";
 import type { ComponentWithChildren } from "types/component-with-children";
+import { cn } from "../../../@/lib/utils";
```

Comment:
go through and remove all the relative paths as possible please

Reply:
We still have Chain and ChainMetadata (v4, v5) chain objects everywhere

Reply:
Updated